src/Deprecated/Conversion/ProjectFileConverter.cs

```diff
@@ -2629,7 +2627,6 @@ private static ProjectItemElement ConvertClassicComReference(XmlElementWithLocat
         /// <returns></returns>
         private ProjectItemElement ConvertProjectToProjectReference(XmlElementWithLocation referenceElement, ProjectItemGroupElement referencesItemGroup, string referenceName, ref string referencedProjectGuid)
         {
-            ProjectItemElement newReferenceItem;
```

Comment:
This one is unsafe to delete. When this function returns you can just return `referencesItemGroup.AddItem(XMakeProjectStrings.projectReference, ProjectCollection.Escape(value));`

Reply:
This is actually part of why I like this change: mistakes like this cause compile-time errors instead of execution-time errors. It looks like, in this case, elachlan refactored it correctly, but when merging in master, overwrote the change you suggested without overwriting removing the declaration, hence the problem. Should be easy enough to fix.